src/dash-table/dash/DataTable.js

```diff
@@ -79,12 +83,18 @@ export const defaultProps = {
     virtualization: false
 };
 
+const cellCoordsIDsProp = PropTypes.exact({
+    row: PropTypes.number,
+    column: PropTypes.number,
+    row_id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    column_id: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
+});
+
 export const propTypes = {
     /**
-     * The [row, column] index of which cell is currently
-     * active.
+     * The row and column indices and IDs of the currently active cell.
      */
-    active_cell: PropTypes.array,
+    active_cell: cellCoordsIDsProp,
```

Comment:
Assuming the resulting metadata.json is correct b/c the value is defined in the same file?

Reply:
oh good call... looks like I can't 🌴 this in the existing `react-docgen` implementation ☹️

Reply:
back to explicit prop types -> https://github.com/plotly/dash-table/pull/412/commits/397b914aa42c84f15ac513a905f2fb1b5e71b917